conda_smithy/templates/run_docker_build.tmpl

```diff
@@ -26,9 +26,12 @@ show_channel_urls: true
 CONDARC
 )
 
+HOST_USER_ID=$( hash docker-machine 2>/dev/null && docker-machine ssh $(docker-machine active) 'id -u $(whoami)' || id -u $(whoami) )
```

Comment:
I've just tested running docker as root. And it actually works, because the ```-o``` in ```useradd --shell /bin/bash -u $USER_ID -o -c "" -m conda``` allow to create an user with a duplicated id.
So the ```conda``` user is created with id 0. I did not find any problem this could cause, so for simplicity I vote to leave it like this.